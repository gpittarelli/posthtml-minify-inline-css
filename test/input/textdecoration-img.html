<a href="" style="color: red;text-decoration: none;font-size:20px;"><img src="" alt="Something" style="vertical-align: middle;text-decoration: none;font-size:20px;"></a>
