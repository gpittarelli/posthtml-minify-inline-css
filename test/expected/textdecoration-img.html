<a href="" style="text-decoration: none;"><img src="" alt="Something" style="vertical-align: middle;text-decoration: none;font-size:20px;"></a>
